elasticmagic_qf_attrs/facet.py

```diff
@@ -421,8 +421,9 @@ def _apply_agg(self, search_query: SearchQuery) -> SearchQuery:
                 },
             )
             min_max_filters = [
-                f for f in filters
-                if f.field is not self.field
```

Comment:
turns out not every filter has `field` property, for example `elasticmagic.expression.Bool`

so I had to change this condition